### DISCUSSION TITLE
Where are the app settings filed stored on disk? I want to back them up.

### DISCUSSION THREAD

**Discussion by freddiehaddad** [FancyWM/fancywm#278] [Q&A] [answered]
I want to back up my FancyWM settings.  Where are the settings files stored?

**Comment by freddiehaddad** [comment, marked as answer]
Found them:

C:\Users\\[UserName]\AppData\Local\Packages\2203VeselinKaraganev.FancyWM_9x2ndwrcmyd2c\LocalCache\Roaming\FancyWM\

**Comment by veselink1** [comment]
`Settings` > `General` > `Show settings.json in containing folder` is a way to get there through the UI.
